package.json

```diff
@@ -26,13 +26,13 @@
   },
   "homepage": "https://github.com/fastify/fastify-static",
   "dependencies": {
+    "fastify": "^1.3.0",
```

Comment:
`fastify-plugin` does not export the respective `fastify.FastifyPlugin` type, so it needs `fastify-plugin` typings patching. Surely I could add some typings tests, but I think it's a bit pointless for a single re-exported `FastifyPlugin` interface. It would've make  sense if there would be a way to impl a `FastifyReply` type extension/enhancing, but afaik it's impossible in TS right now.

I don't like this neither, @mcollina should I go for a PR to `fastify-plugin` too ?